images/runtime/php-fpm/template.base.Dockerfile

```diff
@@ -78,6 +78,13 @@ RUN pecl install redis && docker-php-ext-enable redis
 # https://github.com/Imagick/imagick/issues/331
 RUN pecl install imagick && docker-php-ext-enable imagick
 
+# deprecated from 5.*, so should be avoided 	
+RUN set -eux; \	
+    if [[ $PHP_VERSION != 5.* && $PHP_VERSION != 7.0.* ]]; then \	
+        echo "pecl/mongodb requires PHP (version >= 7.1.0, version <= 7.99.99)"; \	
```

Comment:
very minor nitpick: this comment/echo statement is no longer true, as we can have mongodb >= 8. Can we remove this or update it to be more accurate?

Reply:
`echo` removed. Thanks!